src/uu/stty/src/stty.rs

```diff
@@ -299,7 +302,78 @@ fn stty(opts: &Options) -> UResult<()> {
                         ));
                     }
                 }
-            // baud rate setting
+            } else if *arg == "line" {
+                match args_iter.next() {
+                    Some(line) => {
+                        if let Ok(n) = line.parse::<u32>() {
```

Comment:
What's the reason for parsing `u32` instead of `u8`? It would allow you to get rid of the size check and the casting.